src/static/css/page.css

```diff
@@ -31,10 +31,18 @@
   margin: 0 -1rem;
 }
 
+.width-20 {
+  width: 20%;
+}
```

Comment:
Is there a mobile media query that overrides table widths to 100%? I can't imagine 20% looking legible on a mobile screen.

Reply:
This is for an individual column not the whole table. If you have a very wide column it greedily forces the first column to be too small.

So like this:

![image](https://user-images.githubusercontent.com/10931297/100645043-0cbd9880-3334-11eb-80cb-402abac9cae4.png)

Instead of this:

![image](https://user-images.githubusercontent.com/10931297/100645095-1cd57800-3334-11eb-8b84-ab40a8fa4495.png)

But to be honest I'm going to use the `no-wrap` utility class added in the PR anyway so can remove the `width-20` if you want. Was more adding it in case I needed it in future as originally thought about that. Or I could change it to min-width?

Anyway this is what the table looks like with `no-wrap` on that first column:

![image](https://user-images.githubusercontent.com/10931297/100645347-73db4d00-3334-11eb-90f7-0371dff99143.png)

Reply:
+1 `no-wrap` seems to do exactly what we want in this case

Reply:
OK I'll just remove the `width-20` one then as not needed.

Reply:
And it's gone!!